packages/react-components/react-dialog/src/components/Dialog/Dialog.cy.tsx

```diff
@@ -1,3 +1,5 @@
+/// <reference types="cypress-real-events" />
```

Comment:
this is breaking master pipeline, please remove this to fix it. ty https://uifabric.visualstudio.com/fabricpublic/_build/results?buildId=341222&view=logs&j=e51d397b-0a7b-5b89-4b48-0052e021371d&t=88c41216-94ff-5ca8-d686-48ce16c1eefc

Reply:
Here's the follow up PR https://github.com/microsoft/fluentui/pull/31230